logging/zerolog_adapter/logger.go

```diff
@@ -79,6 +79,12 @@ func (l Logger) Debug(args ...interface{}) {
 	event.Timestamp().Msg(fmt.Sprint(args...))
 }
 
+func (l Logger) DebugWithError(msg string, err error) {
```

Comment:
why we cannot use builder for this situation?

Reply:
i mean without extra func

Reply:
We can, but it is quite a common case, so I made additional methods for cleaner code

Reply:
we will move on builder in end, won't it? 

or we save two logger components event + simple log, after we drop func is nor use?

Reply:
Removed these method after discussion